COLLADAMax/scripts/COLLADAMax.vcxproj

```diff
@@ -193,6 +225,14 @@
       <Configuration>Release_Max2016</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release_Max2017|Win32">
```

Comment:
This config does not exist (win32 2017) - not a problem for this PR, but I should look into this at later time